### ISSUE TITLE
Skin Cancer Detection

### ISSUE THREAD

**Issue by Sindhu-2004** [abhisheks008/DL-Simplified#596]
A skin cancer detection model using TensorFlow to classify images as malignant or benign.

**Comment by abhisheks008** [comment]
Can you share the dataset and approach for solving this issue?

**Comment by Sindhu-2004** [comment]
Project Title : Skin Cancer Detection
Aim : A skin cancer detection model using TensorFlow to classify images as malignant or benign.
Dataset : 
[Skin Cancer ISIC] https://www.kaggle.com/datasets/nodoubttome/skin-cancer9-classesisic
Approach : This model is implemented using TensorFlow, Convolutional Neural Network, Seaborn, Matplotlib, PIL, scikit-learn

**Comment by abhisheks008** [comment]
Hi @Sindhu-2004 can you please share your deep learning approach a bit briefly? Need some clarification in the architectures/methods/models you are trying to implement here.

**Comment by jeet-Abhi123** [comment]
Hi! @abhisheks008 I would like to do this project.
Dataset : HAM10000 (7 classes of skin lesions) 
Approach : For image preprocessing I will be experimenting with dull razor, k-means or YOLOv4. After that I will be applying Transfer learning techniques like Resnet or Mobilenet and adding my own customized ANN layers after that. Finally I will apply KerasTuner for  Hyperparameters  and plot the graphs for results.

**Comment by abhisheks008** [comment]
> Hi! @abhisheks008 I would like to do this project. Dataset : HAM10000 (7 classes of skin lesions) Approach : For image preprocessing I will be experimenting with dull razor, k-means or YOLOv4. After that I will be applying Transfer learning techniques like Resnet or Mobilenet and adding my own customized ANN layers after that. Finally I will apply KerasTuner for Hyperparameters and plot the graphs for results.

But this issue is raised by another contributor, you can look for the issues which are raised by me.

**Comment by jeet-Abhi123** [comment]
@abhisheks008  But your issues has been assigned to someone else.

**Comment by abhisheks008** [comment]
> @abhisheks008 But your issues has been assigned to someone else.

Not all the issues have been assigned. Check out in the issues section, many of them are not assigned.

**Comment by abhisheks008** [comment]
Along side EfficientNet can you implement 2 more architectures for this project?

**Comment by Sindhu-2004** [comment]
I would like to add architectures like **VGG-16** to develop a model, which contributes to the improvement of accuracy and **ResNet (Residual Neural Network)** which is used to obtain the diagnosis report as a confidence score with high accuracy.

**Comment by abhisheks008** [comment]
Cool then VGG, EfficientNet and ResNet are the final models right? @Sindhu-2004

**Comment by Sindhu-2004** [comment]
Yes sir

**Comment by abhisheks008** [comment]
Assigned @Sindhu-2004
